python/ray/data/dataset.py

```diff
@@ -3000,8 +3005,8 @@ def window(
                 window will still include at least one block. This is mutually
                 exclusive with ``blocks_per_window``.
         """
-        from ray.data.dataset_pipeline import DatasetPipeline
         from ray.data._internal.plan import _rewrite_read_stage
+        from ray.data.dataset_pipeline import DatasetPipeline
```

Comment:
What are the changes in this module? I'm having a hard time finding any outside of the import reordering.

Reply:
It's possible that there was a change here that was later reverted. I'll check